### ISSUE TITLE
workflow: nextflow script

### ISSUE THREAD

**Issue by joschrew** [joschrew/ocrd-webapi-implementation#13]
maybe test param.venv in nextflow-scripts. venv should not be provided by the user it should be hidden to the user as the user doesn't necessarily have to deal with it

**Comment by MehmedGIT** [comment]
Agree. The user should not know the `param.venv`. However, the `venv` requirement comes from the OCR-D software itself. When installing the software locally it is still required to activate the `venv` ([reference](https://ocr-d.de/en/setup#ocrd_all-natively)). I am not sure if there is an easy way to go around that.
